cmd/main.go

```diff
@@ -287,9 +287,10 @@ func main() {
 	if Options.DeployTarget == deploymet_type_k8s {
 		go func() {
 			defer apiEnabler.Enable()
-			//Run first ISO dummy for image pull, this is done so that the image will be pulled and the api will take less time.
-			// blocking function that can take a long time.
-			bminventory.GenerateDummyISOImage(log, generator, eventsHandler)
+			// Upload the live image which will serve as a basis for user-generated images.
+			if err = generator.UploadBaseISO(); err != nil {
```

Comment:
On each update we will have 3 replicas that will upload the iso to the same place. maybe we should consider leader election? or check if we already have a correct version in s3? 
Maybe it's not a real issue for now and can be resolved in another PR.